articles/active-directory/device-management-faq.md

```diff
@@ -85,6 +85,13 @@ For down-level Windows OS versions that are on-premises AD domain-joined:
 2.	Type `"%programFiles%\Microsoft Workplace Join\autoworkplace.exe /l"`.
 3.	Type `"%programFiles%\Microsoft Workplace Join\autoworkplace.exe /j"`.
 
+---
+**Q: How do I unjoin an Azure AD Joined device locally on the device?
+**A:** 
+- For hybrid Azure AD Joined devices, make sure to turn off auto registration so that the scheduled task does not register the device again. Next, open command prompt as an administrator and type `dsregcmd.exe /debug /leave`. Alternatively, this command can be run as a script across multiple devices to unjoin in bulk.
```

Comment:
Please remove the Hybrid Azure AD joined devices from this. Hybrid Azure AD joined should be addressed separately.

Also, can you please add a FAQ for 'ownership transfer' for an AADJ device when the 'owner' leaves the company etc.